### ISSUE TITLE
Support AA Accessibility

### ISSUE THREAD

**Issue by alvarosaburido** [asigloo/vue-dynamic-forms#65]
**Is your feature request related to a problem? Please describe.**
The library lacks correct accessibility.

**Describe the solution you'd like**
Pass AA
